docs/api/browser-window.md

```diff
@@ -1227,6 +1227,16 @@ Returns `String` - The title of the native window.
 **Note:** The title of the web page can be different from the title of the native
 window.
 
+#### `win.setAccessibleTitle(title)`
```

Comment:
please add to `docs/api/modernization/property-updates.md`

Reply:
this is new, so it doesn't need to go there!

Reply:
That said this should definitely be a property `win.accessibleTitle`